src/WatchDogs.Contracts/Contracts.csproj

```diff
@@ -6,4 +6,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="8.0.0" />
```

Comment:
You have installed NuGets for the whole solution, meaning they are in every project in it. However, they are currently only needed in Infrastructure.DxTrade and transitively in Application (because it contains Program.cs and all the DI registration stuff). Please leave remove these packages from `Contracts`.

If you want to install a NuGet package to the specific project, right-click on the project -> Manage NuGet Packages and you are good to go.

Reply:
Okey done.